pillar/master/config.sls

```diff
@@ -1,109 +1,24 @@
-{% set ONE_WEEK = 604800 %}
```

Comment:
This file contains some reactors and proxy settings that are env dependent. Shouldn't those be modified?